src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -32,7 +32,9 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
     this.card = createCardElement(config.card);
     if (this.card) {
       this.appendChild(this.card);
-      this.card.hass = this.hass;
+      if (this.hass) {
+        this.card.hass = this.hass;
```

Comment:
This is not correct, there is no need to set it if it's going to be invisible. We should do `this.hass = this._hass` and trigger our hass setting logic.

Reply:
I think this is the main takeaway, correct? Like we are doing for the stack cards, have a private hass and setter that is applied once and only if visible.

Reply:
yeah.